safe_transaction_service/safe_messages/migrations/0002_alter_safemessageconfirmation_owner.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.1.3 on 2022-11-16 14:35
+
+from django.db import migrations
+
+import gnosis.eth.django.models
+
+
+class Migration(migrations.Migration):
```

Comment:
What is the context for this migration. The model was not changed 🤔

Reply:
I forgot to add the migration in the previous PR. Either way I will reset migrations before going to production when modifying the `description` -> `safeAppId`